### PR TITLE
Add settings service

### PR DESCRIPTION
**Pull request by deleon117**

### CODE REVIEW

File: src/ansys/additive/core/additive.py [hunk 1]
```diff
@@ -160,6 +161,9 @@ def __init__(
             server_connections, host, port, nservers, product_version, self._log, linux_install_path
         )
         self._nsims_per_server = nsims_per_server
+        if self._nsims_per_server > 1:
+            self.apply_server_settings()
```
Comment (pkrull-ansys) [inline src/ansys/additive/core/additive.py:165]: This design requires the Additive object to keep a copy of all the server settings. Now that we've moved concurrent simulation management to the server, is there any reason to keep track of `nsims_per_server`? We can always get the latest server settings by calling the list settings endpoint.

File: pyproject.toml [hunk 2]
```diff
@@ -24,7 +24,7 @@ classifiers = [
 ]
```
Comment (pkrull-ansys) [inline pyproject.toml:24]: Let's bump the version of pyadditive. It doesn't really matter but old habits die hard.

**Review by pkrull-ansys — CHANGES_REQUESTED** [review]

**Comment by pkrull-ansys** [comment]
Also, please update CHANGELOG.md. I keep forgetting to do this.

File: pyproject.toml (hunk 2 above)
Comment (deleon117) [reply pyproject.toml:24]: Bumped.

File: src/ansys/additive/core/additive.py (hunk 1 above)
Comment (deleon117) [reply src/ansys/additive/core/additive.py]: `nsims_per_server` no longer a member or property of `Additive`

File: src/ansys/additive/core/additive.py [hunk 3]
```diff
@@ -159,7 +160,10 @@ def __init__(
         self._servers = Additive._connect_to_servers(
             server_connections, host, port, nservers, product_version, self._log, linux_install_path
         )
-        self._nsims_per_server = nsims_per_server
+
+        initial_settings = {"NumConcurrentSims": str(nsims_per_server)}
+        print(self.apply_server_settings(initial_settings))
```
Comment (pkrull-ansys) [inline src/ansys/additive/core/additive.py:165]: Let's use `LOG.info()` instead of `print`.

File: src/ansys/additive/core/additive.py [hunk 4]
```diff
@@ -259,6 +251,42 @@ def about(self) -> None:
             for server in self._servers:
                 print(server.status())
 
+    def apply_server_settings(self, settings: dict[str, str]) -> dict[str, list[str]]:
+        """Apply settings to each server.
+
+        Current settings include:
+        - number of concurrent simulations per server, defined by nsims_per_server.
```
Comment (pkrull-ansys) [inline src/ansys/additive/core/additive.py:258]:
```suggestion
        - ``NumConcurrentSims``: number of concurrent simulations per server.
```

**Review by pkrull-ansys — APPROVED** [review]
Approved with suggestions.

File: src/ansys/additive/core/additive.py (hunk 3 above)
Comment (deleon117) [reply src/ansys/additive/core/additive.py]: Fixed.

File: src/ansys/additive/core/additive.py (hunk 4 above)
Comment (deleon117) [reply src/ansys/additive/core/additive.py]: Fixed.

**Pull request merged**